Makefile.ibm

```diff
@@ -77,12 +78,26 @@ docker-login:
 	@echo $(DOCKER_PASS_ART) | docker login -u $(DOCKER_USER_ART) --password-stdin $(DOCKER_REGISTRY_ART);
 	@echo $(DOCKER_PASS_ICR) | docker login -u $(DOCKER_USER_ICR) --password-stdin $(DOCKER_REGISTRY_ICR);
 
-docker-publish-images: docker-login
-	for image_name in $(DOCKER_IMAGES_TAGGED) ; do                                             \
-		for registry in $(DOCKER_REGISTRIES) ; do                                         	   \
-			$(MAKE) docker-publish-image                                              		   \
-			IMAGE_NAME=$${image_name} DOCKER_REGISTRY=$${registry}; 						   \
-		done                                                                              	   \
+docker-publish-images:
```

Comment:
```suggestion
docker-publish-images: docker-login
```

Reply:
dang ... I thought I put that back; took it out for testing... go catch!

Reply:
added docker-login prereq to docker-publish-images: target